src/test/java/com/revature/studyforce/notification/service/SubscriptionServiceTest.java

```diff
@@ -0,0 +1,84 @@
+package com.revature.studyforce.notification.service;
+
+import com.revature.studyforce.notification.model.Subscription;
+import com.revature.studyforce.notification.repository.SubscriptionRepository;
+import com.revature.studyforce.user.model.User;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.TestPropertySource;
+import static org.mockito.ArgumentMatchers.any;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+@SpringBootTest
+@TestPropertySource(locations = "classpath:test-application.properties")
+public class SubscriptionServiceTest {
```

Comment:
@Author Documentation required and @link to service layer it is testing.

Reply:
fixed